package.json

```diff
@@ -21,8 +21,9 @@
     "dev:landing": "lerna run dev --parallel --scope '@blueprintjs/{core,landing-app}'",
     "dev:select": "lerna run dev --parallel --scope '@blueprintjs/{core,select,docs-app}'",
     "dev:table": "lerna run dev --parallel --scope '@blueprintjs/table-dev-app'",
+    "dev:timezone": "lerna run dev --parallel --scope '@blueprintjs/{core,select,timezone,docs-app}'",
```

Comment:
Why select?

Reply:
I would just skip this command / it's not very useful (I prefer running the scripts in separate terminals) / we can add it back if anyone actually wants it